docs/reference/field-types/dateAndTime.md

```diff
@@ -1,12 +1,14 @@
 # `dateAndTime`
 
-`dateAndTime` fields are text field with UI support, and limitation, for saving date and time values. Date and times are stored in ISO 8601 format. `YYYY-MM-DDTHH:MM:SSZ` format.
+`dateAndTime` fields are text field with UI support, and limitation, for saving date and time values. Dates and times are stored in ISO 8601 format, for example: `2022-01-01T03:00:00Z`.
```

Comment:
I think it would be better for the first sentence to just say:

"`dateAndTime` fields provide subfields for entering date and time values in a user-friendly way."